docs/mesos/ManualInstallCalicoCNI.md

```diff
@@ -0,0 +1,39 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.19.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Calico-CNI with the Unified Containerizer - Manual Install Guide
```

Comment:
Do we know what the right term is here? Unified containerizer vs Mesos containerizer?